sdk/eventgrid/test-resources.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "String"
+    }
+  },
+  "variables": {
+    "apiVersion": "2020-04-01-preview",
```

Comment:
I'm guessing we shouldn't use a preview API version here? Can probably use the most recently released one which seems to be `2020-06-01`.